Keas.Mvc/Controllers/Api/KeysController.cs

```diff
@@ -91,7 +89,7 @@ public async Task<IActionResult> List()
 
         // list all keys for a space
         [HttpGet]
-        [ProducesResponseType(typeof(IEnumerable<Key>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(IEnumerable<KeyInfo>), StatusCodes.Status200OK)]
         public async Task<IActionResult> GetKeysInSpace(int spaceId)
```

Comment:
@wongband This returns:

{
    "key": {
      "code": "AEXMPLE2",
      "keyXSpaces": [
        {
          "id": 23,
          "keyId": 3562,
          "space": null,
          "spaceId": 3637
        }
      ],
      "serials": [
        {
          "number": "1",
          "status": "Destroyed",
          "keyId": 3562,
          "keySerialAssignment": null,
          "keySerialAssignmentId": null,
          "title": "1",
          "id": 8595,
          "name": "1",
          "notes": "asd",
          "tags": "",
          "team": null,
          "teamId": 21,
          "active": true
        },
        {
          "number": "Test2",
          "status": "Active",
          "keyId": 3562,
          "keySerialAssignment": null,
          "keySerialAssignmentId": null,
          "title": "Test2",
          "id": 8596,
          "name": "Test2",
          "notes": "",
          "tags": "",
          "team": null,
          "teamId": 21,
          "active": true
        }
      ],
      "title": "Destroyed key. Delete this line.",
      "id": 3562,
      "name": "Destroyed key. Delete this line.",
      "notes": "asdf",
      "tags": "Imported",
      "team": null,
      "teamId": 21,
      "active": true
    },
    "id": 3562,
    "serialsTotalCount": 1,
    "serialsInUseCount": 0,
    "spacesCount": 0
  }